synapse/handlers/sync.py

```diff
@@ -185,6 +185,7 @@ def __nonzero__(self):
 class SyncHandler(object):
 
     def __init__(self, hs):
+        self.hs = hs
```

Comment:
We try and avoid storing the `hs` object itself, to avoid the temptation to do `hs.get_foo()` in the code. Can this be `self.config = hs.config` please?

Reply:
I made it `hs_config` (seemed clearer to me)